sklearn/decomposition/fastica_.py

```diff
@@ -300,7 +300,9 @@ def g(x, fun_args):
         n_components = min(n, p)
     if (n_components > min(n, p)):
         n_components = min(n, p)
-        warnings.warn('n_components is too large: it will be set to %s' % n_components)
+        warnings.warn(
```

Comment:
please revert formatting changes.

Reply:
I run `pycodestyle` to autoformat the code. Apparently, that line was changed by it. I couldn't found in the documentation of the stable version how to format the code. Is there an automatic way to do the formatting?

Got it. I'll revert the changes in the next PR.

Reply:
We follow [PEP8](https://www.python.org/dev/peps/pep-0008/). Some editors/IDEs give suggestions and warnings as you code, but the configurations depends on the editor you use, and not included in the contributing guides.

You can always run `flake8` on your code to see if there are any issues.